rebar.config.script

```diff
@@ -7,7 +7,8 @@ case erlang:function_exported(rebar3, main, 1) of
         %% profiles
         [{deps, [{meck, ".*",{git, "https://github.com/eproxus/meck.git", {tag, "0.8.4"}}},
                  {jsx, ".*", {git, "git://github.com/talentdeficit/jsx.git", {tag, "2.8.0"}}},
-                 %% {hackney, ".*", {git, "git://github.com/benoitc/hackney.git", {tag, "1.2.0"}}},
+                 {unicode_util_compat, ".*", {git, "https://github.com/benoitc/unicode_util_compat.git", {tag, "0.3.1"}}},
+                 {hackney, ".*", {git, "git://github.com/benoitc/hackney.git", {tag, "1.10.1"}}},
```

Comment:
are we sure we still desperately need hackney for GD endpoints?

Reply:
`lhttpc` doesn't play well with TLS1.2 used by AWS.